docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -318,10 +317,34 @@ eigenlayer operator status operator.yaml
 
 ---
 
-### Metadata Updates
+## Metadata Updates
 
 This is the command you can use to make changes or updates to the metadata of your operator.
 
 ```
 eigenlayer operator update operator.yaml
 ```
+
+
+
+## delegationApprover Design Patterns
+
+Delegation Approver functionality can be used in multiple ways to give Operators additional programmatic control over which Restakers they accept delegation from.
+
+
+### EOA Signature Approval
+
+One series of designs involves passing a unique signature from the Operator to the Restaker requesting approval. The unique signature will have a corresponding ‘salt’ (unique value used once) and an ‘expiry’. The Restaker passes the signature (salt & expiry) into the `DelegationManager.delegateTo` function ([source here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/mainnet/src/contracts/core/DelegationManager.sol#L135-L155)). This function uses EIP1271 to check the signature, so either:
+- A) The Operator has set an EOA as their `delegationApprover` and the DelegationManager simply checks that the signature is a valid ECDSA signature from the EOA.
```

Comment:
it might help to explain more about how this would work in practice. 
in all likelihood, you would have a signing process running with the `delegationApprover` key, which would receive delegation requests from Stakers, check them against some criteria, and issue signatures which are passed back to these Stakers. perhaps this is obvious though.

Reply:
I'm open to any changes you'd like. Would you care to recommend revised wording for that scenario? Otherwise, we can hold off for now and see how many questions we get on this topic from users.

Reply:
Yeah that's fine. We can explain more if we get a lot of user questions. will resolve this thread